msu/hooks/entity/tactical/player.nut

```diff
@@ -30,4 +39,18 @@
 			}
 		}
 	}
+
+	local onSerialize = o.onSerialize;
+	o.onSerialize = function( _out )
+	{
+		this.getFlags().set("LevelUpsSpent", this.m.LevelUpsSpent);
+		onSerialize(_out);
+	}
+
+	local onDeserialize = o.onDeserialize;
+	o.onDeserialize = function( _in )
+	{
+		onDeserialize(_in);
+		this.m.LevelUpsSpent = this.getFlags().get("LevelUpsSpent");
```

Comment:
I just realized this will default to false which is probably not ideal, you should add a `has` check and set it to 0 if false.

Reply:
Good catch. Will do.